pilot/pkg/proxy/envoy/discovery_test.go

```diff
@@ -648,6 +723,14 @@ func TestListenerDiscoverySidecar(t *testing.T) {
 			name: "egress-rule-tcp",
 			file: egressRuleTCP,
 		},
+		{
+			name: "egress-rule",
+			file: externalServiceRule,
+		},
+		{
+			name: "egress-rule-tcp",
```

Comment:
nit: should the names be changed to external-service and external-service-tcp?

Reply:
the name is the same as egress so that we can share the output golden file with egress. we can create a new golden file instead, if you think that would be better.

Reply:
I see, good, maybe it should be documented.